docs/index.rst

```diff
@@ -1,22 +1,21 @@
-####################
-Carsus Documentation
-####################
+**********************************
+Welcome to Carsus's documentation!
+**********************************
 
-Carsus is a TARDIS support package for creating and working with atomic datasets
+Carsus is the `TARDIS <http://tardis-sn.github.io/tardis/>`_ support package for creating atomic datasets.
```

Comment:
I would say that "Carsus is a package to manage atomic datasets. It can read data from a variety of sources and output them to file formats readable by radiative transfer codes."

Reply:
Ok!